src/utils/encryption.ts

```diff
@@ -0,0 +1,44 @@
+import * as crypto from "crypto";
+
+/**
+ * Encrypts a secret using AES-256-CBC encryption
+ *
+ * @param secret - The secret to be encrypted
+ * @param key - The encryption key - a 32-byte string
+ * @returns The encrypted secret along with the initialization vector (IV), separated by a colon
+ */
+export function encrypt(secret: string, key: string): string {
```

Comment:
Inspiration / instruction for these functions came from here - https://www.geeksforgeeks.org/node-js-crypto-createcipheriv-method/

I had originally planned to use a library for this, but it looks like the move is towards doing this natively in Node - `[crypto-js](https://www.npmjs.com/package/crypto-js)` is no longer maintained and advises using the Node crypto module directly.